### ISSUE TITLE
provide a way to go "back" when digging through multiple files via ctrl+clicks on methods/functions.

### ISSUE THREAD

**Issue by phillipkerman** [Microsoft/vscode#57904]
Copy Eclipse's "Last Edit Location (Ctrl+Q)" and "Back to <file> (Ctrl+Left).

Alt+Left Arrow doesn't jump back more than one level.  If I go into file a, ctrl+click a method to navigate to file b, then ctrl+click to navigate to file c... then Alt+Left Arrow.... I go back only to file b... never to a.

I've asked many vscode power users and advocates and have found no alternative.  Thanks!.

**Comment by bpasero** [comment]
@phillipkerman can you not press Alt+Left twice to go back? Or do you want a way to go back to the last edit location?

**Comment by phillipkerman** [comment]
When I Alt+Left, I can go back only one-level.  That is, if I ctrl+click to navigate out of one file, I get the codeLens thing... and I CAN ctrl+click in that to go to a third file.... but at that point, alt+left behaves like tab left through that third file.  I want to go back to the second file, then back to the first AND have my edit position.  See this clip of the exact behavior in eclipse: 

![out](https://user-images.githubusercontent.com/543856/45507003-c32e0680-b745-11e8-98e2-035189b148cd.gif)

**Comment by bpasero** [comment]
@phillipkerman can you show me the behaviour in VSCode also? Alt+Left to go back should work

**Comment by phillipkerman** [comment]
In this clip, I use Alt+Left after I land in the Deck file.  It just moves me up like ctrl+left arrow.  
Also, I believe I turned off codeLens (which I sort of liked but thought maybe it could be a conflict here).  The new non-json settings thing has got me a bit lost--so I am happy to dig through my settings.  (I included a screenshot in the video for you to see I had 'nothing up my sleeves'.)

![out](https://user-images.githubusercontent.com/543856/45564479-609c3f80-b805-11e8-9169-2b81453cfede.gif)
